.github/workflows/changelog.yml

```diff
@@ -28,11 +28,12 @@ jobs:
       - run: pip install towncrier
       - run: towncrier check
       - run: towncrier build --draft | grep -P '#${{ github.event.number }}'
-  changelog_uses_towncrier:
+  prevent_manually_editing_changlog:
     if: ${{ !contains(github.event.pull_request.labels.*.name, 'allow-manual-changelog-edit') }}
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - run: git diff HEAD ${{ github.event.pull_request.base.sha }} --no-patch --exit-code CHANGES.rst
+      - name: prevent direct changes to `CHANGES.rst` (you can override this with the `allow-manual-changelog-edit` label)
```

Comment:
It's nice to have more text here; maybe it's too long, but:
prevent direct changes to CHANGES.rst (use towncrier instead; override this with the allow-manual-changelog-edit label)
If you think.  I'll go ahead and approve anyway.

Reply:
```suggestion
      - name: prevent direct changes to `CHANGES.rst` (write a towncrier fragment in `changes/` instead; you can override this with the `allow-manual-changelog-edit` label)
```